CHANGELOG.rst

```diff
@@ -2,12 +2,30 @@
 Changelog
 *********
 
+1.0.4 -- 2018-05-xx
```

Comment:
It's not ideal, but with the travel and everything in play, if you want to put a reasonable date in here I can commit to getting it pushed by then, depending on your thoughts on the rest of the comments.